templates/projects/tasks.html

```diff
@@ -21,6 +21,12 @@
             {{ render_project_card_option(project, upload_method, title=_('Export Tasks'), explanation=_('Export tasks to JSON and CSV'), link=url_for('project.export_to', short_name=project.short_name), link_action_text=_('Export'), icon='cloud-download')}}
         </div>
         {% endif %}
+        <!-- Project owner, subadmins that are project co-owners and admins can delete tasks to project -->
+        {% if  (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
```

Comment:
Project owner should be able to delete tasks without a subadmin privilege, right?

Reply:
A regular user can also be the project owner. hence only owners submin access are entitled to delete tasks.

Reply:
I felt like the permissions would be similar to the use case of importing tasks on line 13

Reply:
ok